core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/ShardingContainerPoolBalancer.scala

```diff
@@ -265,6 +268,9 @@ class ShardingContainerPoolBalancer(
     chosen
       .map { invoker =>
         setupActivation(msg, action, invoker)
+        logging.info(
+          this,
+          s"activation ${msg.activationId} of $actionType-action '${msg.action.asString}' invoked by namespace '${msg.user.namespace.name.asString}' with memory-limit of ${action.limits.memory.megabytes} MB will be executed on invoker${invoker.toInt}")
```

Comment:
small tweak suggested:

```
s"activation ${msg.activationId} for '${msg.action.asString}' by namespace
'${msg.user.namespace.name.asString}' with memory limit 
${action.limits.memory.megabytes}MB assigned to $actionType $invoker"
```

Reply:
While I like your proposal, we should probably not print the action type (managed or blackbox) in a way that it implies it's the invoker type with the background that we have discussions to drop the separation between managed / blackbox invokers. So I suggest that the message should express that the action is either managed or blackbox.

Reply:
Fair although that discussion hasn't resolved and the message is accurate for today's status. Tweaking further for your input nonetheless.

```
s"activation ${msg.activationId} for '${msg.action.asString}' ($actionType) 
by namespace '${msg.user.namespace.name.asString}' with memory limit 
${action.limits.memory.megabytes}MB assigned to $invoker"
```